addons/sms/static/src/js/fields_phone_widget.js

```diff
@@ -63,7 +63,7 @@ Phone.include({
             });
             $composerButton.prepend($('<i>', {class: 'fa fa-mobile'}));
             $composerButton.on('click', this._onClickSMS.bind(this));
-            this.$el = $('<div/>').append(this.$el).append($composerButton);
+            this.$el.html($('<div/>').append(this.$el.clone()).append($composerButton));
```

Comment:
You can replace with: `this.$el.append($composerButton);`

Please test it again after but it seems to work fine on my side.

Reply:
There is a small difference between your solution and ours : 
- In your case: when the mouse hover the link of SMS, the link color of the phone change also.
- In our case: the phone link not changed. (seems more correct)
Then I think that we wants this version ?

Reply:
Then maybe use `this.$el.after($composerButton);` ?

Because right now your structure looks like this:
![Selection_314](https://user-images.githubusercontent.com/41282536/67861662-f043a100-fb20-11e9-9640-ed60f89a8a2c.png)

Which is a link containing itself, and that a little strange.

Reply:
Unfortunately, It doesn't work in the form view. :/